elasticsearch5/src/main/java/org/vertexium/elasticsearch5/ElasticsearchSearchQueryBase.java

```diff
@@ -513,6 +525,43 @@ protected IdStrategy getIdStrategy() {
         return createIterable(response, filterParameters, hitsIterable, response.getTookInMillis(), hits);
     }
 
+    private List<ElasticsearchVertex> getElasticsearchVertices(SearchHits hits, FetchHints fetchHints, Authorizations authorizations) {
+        return StreamUtils.stream(hits)
```

Comment:
I'm usually not a fan of static imports except for this, I usually static import `StreamUtils.stream` but doesn't matter to much to me